zeq/hbp/detail/imageRawRGBA8.fbs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015, Human Brain Project
+//                     Grigori Chevtchenko <grigori.chevtchenko@epfl.ch>
+
+namespace zeq.hbp;
+
+table ImageRawRGBA8
```

Comment:
ImageRGBA8 is not a separate type, it is a superset of all images of ImageRGBA8 format which can be any image format (png,jpg ...). To me, without the Raw word, it is not informative enough to get what the class is holding.